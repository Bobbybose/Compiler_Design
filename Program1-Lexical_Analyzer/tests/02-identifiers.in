abc
abc123
abc_
abc_123
abc123_

_
_abc
_123
_abc123
_123abc