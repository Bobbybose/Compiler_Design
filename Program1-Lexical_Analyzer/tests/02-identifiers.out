identifier: abc: 1, 1
identifier: abc123: 2, 1
identifier: abc_: 3, 1
identifier: abc_123: 4, 1
identifier: abc123_: 5, 1
identifier: _: 7, 1
identifier: _abc: 8, 1
identifier: _123: 9, 1
identifier: _abc123: 10, 1
identifier: _123abc: 11, 1